SR2025/2024-10-07-tracking-links.md

```diff
@@ -0,0 +1,16 @@
+---
+to: Student Robotics 2025 teams whose kit has been dispatched
+subject: Your kit is on its way!
+---
+
+Hello,
+
+We've dispatched your kit, and it's on its way to you!
+
+The kits have been dispatched with UPS, and should be with you in the next few days. If you haven't received your kit by next week, please let us know!
+
+You can track your shipment by going to [UPS' website](https://www.ups.com/gb/en/Home.page) and entering your tracking code: {{ tracking_code }}.
```

Comment:
minor: is there a reason we can't give a url direct to the tracking? It looks like the form on the homepage is just a GET form, so I think we could template the url fine?

Reply:
We could, but if the schema changes it's more annoying. Putting the code in verbatim isn't that much extra work for users, especially as we link right to the homepage.